charts/templates/ingress.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Values.ingress.name }}
+  annotations:
+    kubernetes.io/ingress.class: nginx
```

Comment:
`kubernetes.io/ingress.class` is deprecated, shall we make `ingressClassName` available as well?

Reply:
Thank you for the feedback!
I made ingress annotations set on `values.yaml` so that annotations can be configurable.